packages/erc-20-lib/src/token-address.js

```diff
@@ -1,12 +1,12 @@
 'use strict'
 
-const { tokens } = require('@uniswap/default-token-list')
+const { findTokenBySymbol } = require('token-list')
 
-const tokenAddress = function (symbol, extraTokens = []) {
-  const tokenData = tokens.concat(extraTokens)
+// TODO remove the default chain id from here
```

Comment:
does this require an issue?
and given fork is hemi related, should default value be hemi chain Id?